commands/raxmon-monitoring-zones-update

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import optparse
+
+from raxmon_cli.common import run_action
+from raxmon_cli.utils import instance_to_dict
+
+OPTIONS = [
+    [['--id'], {'dest': 'id', 'help': 'Entity id'}],
```

Comment:
It's the same for all the other objects (like entities, checks etc). Just following the convention.